resources/views/waitingrooms/partials/_form_waitingrooms.blade.php

```diff
@@ -2,7 +2,7 @@
 
 <div class="form-item -padded">
     {{ Form::label('contest_id', 'Contest ID:', ['class' => 'field-label']) }}
-    {{ Form::text('contest_id', NULL, ['class' => 'text-field']) }}
+    {{ Form::text('contest_id', NULL, ['class' => 'text-field', 'readonly' => 'readonly']) }}
```

Comment:
not a big deal, just curious cause as you said some of the other html properties dont require it